fix: partfuncfmt error

gagan-aryan opened the pull request:
### Description
<!-- Provide a general description of what your pull request does. -->

Previously the program crashed with the following errors 

```
  File "/home/gaganaryan/Desktop/radis/radis/lbl/base.py", line 3157, in _get_parsum
    return self.get_partition_function_interpolator(molecule, iso, state)
  File "/home/gaganaryan/Desktop/radis/radis/lbl/loader.py", line 2233, in get_partition_function_interpolator
    assert isinstance(parsum, RovibParFuncTabulator)
AssertionError
```
when `parfuncfmt` wasn't passed into the databank. This pr addresses this issue and removes this behavior by using `hapi` as the default partition function.  

Fixes #78

erwanp commented:
Hello @gagan-aryan , well done ! 

Can you also add some comment about HAPI being used as default (if nothing given) in the user functions ? For instance : 
- in [load_databank()](https://github.com/radis/radis/blob/be5f72d197358f2d275ed88634db4b23e98c909e/radis/lbl/loader.py#L693) 
- same [here](https://github.com/radis/radis/blob/be5f72d197358f2d275ed88634db4b23e98c909e/radis/lbl/loader.py#L820)

If you want, I also found a mention of it in the docs in the [lbl.rst](https://github.com/radis/radis/blob/be5f72d197358f2d275ed88634db4b23e98c909e/docs/lbl/lbl.rst) file 
```
parfuncfmt: cdsd                 #  'cdsd', 'hapi', etc.
                                 # format to read tabulated partition function
                                 # file. If `hapi`, then HAPI (HITRAN Python
                                 # interface) is used to retrieve them (valid if
                                 # your databank is HITRAN data). HAPI is embedded
                                 # into RADIS. Check the version.
```

erwanp commented:
Also note that #80 is quite similar and could be fixed in the same PR : two birds one stone !

gagan-aryan commented:
Hey @erwanp . I have added another commit that should resolve #80 . 
I have also updated a few comments and a few other lines in [lib.rst](https://github.com/radis/radis/blob/be5f72d197358f2d275ed88634db4b23e98c909e/docs/lbl/lbl.rst)

erwanp commented:
Well done, merging tomorrow !
